src/python/pants/engine/mapper.py

```diff
@@ -161,18 +156,18 @@ class ResolveError(MappingError):
 class AddressMapper:
   """Configuration to parse build files matching a filename pattern."""
   parser: Parser
-  build_patterns: Tuple
-  build_ignore_patterns: Tuple
-  exclude_target_regexps: Tuple
-  subproject_roots: Tuple
+  build_patterns: Tuple[str, ...]
+  build_ignore_patterns: Tuple[str, ...]
+  exclude_target_regexps: Tuple[str, ...]
+  subproject_roots: Tuple[str, ...]
```

Comment:
This is the only one I couldn't validate through runtime inspection because I have no idea what subprojects are?